### ISSUE TITLE
Unable to determine forwarding port for API "monitoring"

### ISSUE THREAD

**Issue by eprop-aaron** [localstack/localstack#2571]
# Type of request: This is a ...

[x ] bug report
[ ] feature request

# Detailed description

Not able to use the edge service to write metric data to cloudwatch via the javascript aws-sdk.

However, It **DOES** work when using the deprecated cloudwatch port.

## Expected behavior

Metric data to be visible when calling `awslocal cloudwatch list-metrics`

## Actual behavior

404 from edge service. No metric data is persisted, localstack instance logs the following info:

```
INFO:localstack.services.edge: Unable to determine forwarding port for API "monitoring" - please make sure this API is enabled via the SERVICES configuration
```

# Steps to reproduce

Run JS code in "Client code" section.
See 404 Error

## Command used to start LocalStack

`TMPDIR=/tmp/localstack docker-compose up`

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

```js
const AWS = require('aws-sdk')
const cw = new AWS.CloudWatch({
  endpoint: 'http://localhost:4566',
})

const data = {
  Namespace: 'ns',
  MetricData: [
    {
      MetricName: 'metric-name',
      Timestamp: new Date(),
      Value: 12345,
      Unit: 'Milliseconds',
      Dimensions: [
        { Name: 'Environment', Value: 'local' },
      ]
    }
  ]
}

cw.putMetricData(data, (err, data) => {
  if (err) {
      console.log('Error', err)
  } else {
      console.log('Success', JSON.stringify(data))
  }
})
```

**Comment by whummer** [comment]
Thanks for reporting @eprop-aaron . This should be fixed in #2581 - can you please give it a try with the `latest` version of the Docker image? Please report here if the problem persists. Thanks

**Comment by eprop-aaron** [comment]
@whummer I can now confirm this is working as expected with the latest Docker image. Thank you very much 👍
